odoo/models.py

```diff
@@ -3633,6 +3633,26 @@ def get_base_url(self):
             raise ValueError("Expected singleton or no record: %s" % self)
         return self.env['ir.config_parameter'].sudo().get_param('web.base.url')
 
+    def _parse_companies(self, companies):
+        if companies is None:
+            return self.env.company.ids
```

Comment:
I strongly advocate against this kind of fallback, as it often hides programming errors.
The argument should be explicit IMHO.